docs/docs/guide/_sections/binders/two-way.md

```diff
@@ -3,27 +3,28 @@ Two-way binders, like one-way binders, can update the DOM when a model property
 In order to update the model when the user interacts with the DOM, you need to tell Rivets.js how to bind and unbind to that DOM element to set the value on the model. Instead of defining the binder as a single function, two-way binders are defined as an object containing a few extra functions.
 
 ```javascript
-rivets.binders.toggle = {
-  bind: function(el) {
-    adapter = this.config.adapters[this.key.interface]
-    model = this.model
-    keypath = this.keypath
-
-    this.callback = function() {
-      value = adapter.read(model, keypath)
-      adapter.publish(model, keypath, !value)
-    }
+rivets.binders['mobi-calendar'] = {
```

Comment:
Toggle example was buggy as explained in #634 
I found the calendar example more explicit for a two way binder as it used all API including `getValue`. 
Toggle example is be more generic but it has to be reworked.